### PR TITLE
Fix #4118 - remove invalid statement

### LINKED ISSUE #4118: Unregister-PSRepository PSGallery works
[Enter feedback here]

Unregister-PSRepository PSGallery works even though this document says it doesnt, we register an internal repository and unregister PSGallery because our servers do not have direct internet access

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5a218f1d-349d-ad8f-0975-2a5e20520d88
* Version Independent ID: 21624df7-b69b-6ed5-320b-d9274eaa501d
* Content: [Unregister-PSRepository (PowerShellGet)](https://docs.microsoft.com/en-us/powershell/module/powershellget/unregister-psrepository?view=powershell-6)
* Content Source: [reference/6/PowerShellGet/Unregister-PSRepository.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/PowerShellGet/Unregister-PSRepository.md)
* Product: **powershell**
* Technology: **powershell-cmdlets**
* GitHub Login: @edyoung
* Microsoft Alias: **edyoung**

### PR DESCRIPTION
**Pull request by sdwheeler**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[6bee59e](https://github.com/sdwheeler/PowerShell-Docs/pull/4181/commits/6bee59e3f1212fce9c8c2abc7f7dda04dac65f5f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.0/PowershellGet/Unregister-PSRepository.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4118/reference/5.0/PowershellGet/Unregister-PSRepository.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/unregister-psrepository?view=powershell-5.0&branch=pr-en-us-4181) (powershell-5.0) |
[reference/5.1/PowershellGet/Unregister-PSRepository.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4118/reference/5.1/PowershellGet/Unregister-PSRepository.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/unregister-psrepository?view=powershell-5.1&branch=pr-en-us-4181) (powershell-5.1) |
[reference/6/PowerShellGet/Unregister-PSRepository.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4118/reference/6/PowerShellGet/Unregister-PSRepository.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/unregister-psrepository?view=powershell-6&branch=pr-en-us-4181) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C19%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201904192041517767-4181%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=BTJNq0EUoaeMR8hzFW%2FdJO424QtbyErWR1fCXaQsnIw%3D&st=2019-04-19T20%3A41%3A25Z&se=2019-05-20T20%3A46%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/5.0/PowershellGet/Unregister-PSRepository.md [hunk 1]
```diff
@@ -20,20 +20,25 @@ Unregister-PSRepository [-Name] <String[]> [<CommonParameters>]
...
+
+### Example 2: Unregister all repositories
+
+This example uses `Get-PSRepository` to get all registered repositories, and uses the pipeline operator to pass them to `Unregister-PSRepository` to unregister them.
+
+```powershell
+Get-PSRepository | Unregister-PSRepository
 ```
 
 This command uses Get-PSRepository to get all registered repositories, and uses the pipeline operator to pass them to **Unregister-PSRepository** to unregister them.
```
Comment (DCtheGeek) [inline reference/5.0/PowershellGet/Unregister-PSRepository.md]: With the description above the example, this line should be removed.

Comment (sdwheeler) [reply reference/5.0/PowershellGet/Unregister-PSRepository.md]: You couldn't unregister PSGallery in that version of PowerShellGet. That is why I left it.

Comment (DCtheGeek) [reply reference/5.0/PowershellGet/Unregister-PSRepository.md]: Wasn’t clear.  The nearly same paragraph is now above and below that sample. Only need one of them.

Comment (sdwheeler) [reply reference/5.0/PowershellGet/Unregister-PSRepository.md]: Ooops. Missed cleaning up that paragraph. Fixed now.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[4d63ae3](https://github.com/sdwheeler/PowerShell-Docs/pull/4181/commits/4d63ae35fe48e1a5185d9500cc99a93f311211cc)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.0/PowershellGet/Unregister-PSRepository.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4118/reference/5.0/PowershellGet/Unregister-PSRepository.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/unregister-psrepository?view=powershell-5.0&branch=pr-en-us-4181) (powershell-5.0) |
[reference/5.1/PowershellGet/Unregister-PSRepository.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4118/reference/5.1/PowershellGet/Unregister-PSRepository.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/unregister-psrepository?view=powershell-5.1&branch=pr-en-us-4181) (powershell-5.1) |
[reference/6/PowerShellGet/Unregister-PSRepository.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4118/reference/6/PowerShellGet/Unregister-PSRepository.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/unregister-psrepository?view=powershell-6&branch=pr-en-us-4181) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C20%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201904201721081844-4181%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=dkhfHhwqh3%2BLJ1MLR8YqWgeO5%2FqUUMcP%2BsxWZVyNZJo%3D&st=2019-04-20T17%3A20%3A56Z&se=2019-05-21T17%3A25%3A56Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**